demographics.html

```diff
@@ -65,45 +67,48 @@ <h1>Demographics</h1>
             <div class="row">
                 <div class="col-xs-6 col-xs-offset-3 col-sm-4 col-sm-offset-4">
                     <div class="dropdown demo-dropdown">
-                        <span>Choose a District</span>
-                        <select id="mapchoice" class="form-control" onchange="myFunction(this)">
-                          <script type="text/javascript" src="js/maplist.js"></script>  <!-- JS function for creation of drop down list of map zone. Just call 
+                        <div>Service Zone</div>
```

Comment:
this is new text overwriting old text. this is also confusing. i'm not sure why this is happening